test/test.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--slower", action='store_true', default=False)
+args = parser.parse_args()
+if args.slower:
+    regions = [
+        "us-east-1",
+        "us-east-2",
+        "us-west-1",
+        "us-west-2",
+        "ap-south-1",
+        "ap-northeast-1",
+        "ap-northeast-2",
+        "ap-northeast-3",
+        "ap-southeast-1",
+        "ap-southeast-2",
+        "eu-central-1",
+        "eu-west-1",
+        "eu-west-2",
+        "eu-west-3",
+        "eu-north-1",
+        "sa-east-1",
+    ]
+else:
+    regions = ["us-west-2"]
+
+for region in regions:
+    command = f"""
+terraform init -upgrade
+terraform apply -auto-approve -var='profile=thunderdome' -var="name=observe-${{USER}}" -var="region={region}"
```

Comment:
Do not hardcode internal information (environments, accounts) in public modules, even in the tests

Reply:
ack